package.json

```diff
@@ -61,6 +61,7 @@
     "eslint-plugin-react": "^7.14.3",
     "husky": "^4.3.0",
     "minimist": "^1.2.0",
+    "postcss": "^7.0.35",
```

Comment:
what is this? are you using it?

Reply:
that package was already installed, the only thing I did was to move it to the development dependencies.

It is the one that takes care, through webpack that you can use module css or scss in the components